src/lib/flags.ts

```diff
@@ -61,6 +62,7 @@ const featureFlags: Record<FeatureName, FeatureFlag> = {
   TREASURE_UPDATES: betaTimeBasedFeatureFlag(new Date("2024-09-16T00:00:00Z")),
   FSL: defaultFeatureFlag,
   NEW_RESOURCES_GE: defaultFeatureFlag,
+  ANIMAL_BUILDINGS: defaultFeatureFlag,
```

Comment:
Until the feature is ready, beta testers will still need access to the current chicken mechanic.

I would suggest using `testnetFeatureFlag` until it is ready to use.

Reply:
Of course 😅